tests/bootstrap_tests/test_manager.py

```diff
@@ -209,6 +209,37 @@ def create_sg_connection(self):
         return self._mockgun
 
 
+class TestPrepareEngine(ShotgunTestBase):
+
+    def setUp(self):
+        super(TestPrepareEngine, self).setUp({"primary_root_name": "primary"})
+
+    def test_prepare_engine(self):
+        """
+        Makes sure that prepare engine works.
+        """
+        mgr = ToolkitManager(_MockedShotgunUser(self.mockgun, "larry"))
```

Comment:
Since we refactored prepare_engine, which wasn't unit tested, I've added a test.

Reply:
When you are done reviewing this file, you can click [here](https://github.com/shotgunsoftware/tk-core/pull/655/files#diff-1c5588644d43fb0a593da7b0cdf6ccf7R1) to jump to the next relevant section.